packages/suite/src/utils/wallet/coinmarket/coinmarketUtils.ts

```diff
@@ -150,28 +147,24 @@ export const getComposeAddressPlaceholder = async (
         case 'bitcoin': {
             // use legacy (the most expensive) address for fee calculation
             // as we do not know what address type the exchange will use
-            const legacy =
-                networksCompatibility.find(
-                    network =>
-                        network.symbol === account.symbol && network.accountType === 'legacy',
-                ) ||
-                networksCompatibility.find(
-                    network =>
-                        network.symbol === account.symbol && network.accountType === 'segwit',
-                ) ||
-                network;
-            if (legacy && device) {
+            const availableAccounts = network.accountTypes;
+            const bip43Path =
+                availableAccounts['legacy']?.bip43Path ??
+                availableAccounts['segwit']?.bip43Path ??
+                network.bip43Path;
```

Comment:
With the new interface, the `network` bears the `normal` account `bip43path`, while the specific accounts such as legacy are in its children (but are not guaranteed for every bitcoin-type network).